hca_ingest/api/ingestapi.py

```diff
@@ -59,32 +60,58 @@ def unset_token(self):
         self.logger.debug(f'Token unset!')
         return self.headers
 
-    def get_headers(self):
-        # refresh token
-        if self.token and not self.headers.get('Authorization'):
-            self.set_token(f'Bearer {self.token}')
-            print(f'Bearer {self.token}')
+    def get(self, url, **kwargs):
+        if 'headers' not in kwargs:
+            kwargs['headers'] = self.get_headers()
```

Comment:
Currently if the caller of `api.get` doesn't specify a headers parameter the headers passed to the session.get will be the result of the `self.get_headers()` method which is initialised in the `__init__`
```ingestapi.py
def __init__(self, url=None, token_manager=None, session=None):
    # snip
    self.headers = self.__get_basic_header()

    def get_headers(self):
        # snipping token stuff
        return self.headers

    @staticmethod
    def __get_basic_header():
        return {'Content-type': 'application/json'}
```

Otherwise the caller is providing their own headers object, which might include a different content-type, which we just pass along.

Are you suggesting that the headers we use should be a mix of our defaults and those provided by the caller?

Reply:
Ok, thanks for this. I am sorry, but I totally missed it.
This looks fine to me.